conferences/views.py

```diff
@@ -21,7 +21,10 @@
 @require_http_methods(['GET'])
 def user_preferences_index(request):
     zosia = get_object_or_404(Zosia, active=True)
-    ctx = {'objects': UserPreferences.objects.filter(zosia=zosia).all()}
+    # TODO: paging?
+    user_preferences = UserPreferences.objects.filter(zosia=zosia).all()
+    user_preferences = sorted(user_preferences, key=lambda x: x.pk)
```

Comment:
It would be better to sort this in db - with`order_by`. There is option in model `Meta` class to set ordering